sw/applications/example_spi_flash_write/main.c

```diff
@@ -367,12 +360,15 @@ int main(int argc, char *argv[])
     }
     PRINTF("triggered\n\r");
 
+    // In simulation there is no need to wait
+    #ifdef USE_SPI_FLASH
```

Comment:
to avoid too many ifdef, please let's wait in simulation as well, can you please revert this change?